services/gitlab/gitlab-coverage.tester.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const { isIntegerPercentage } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Coverage').get('/gitlab-org/gitlab-runner.json').expectBadge({
+  label: 'coverage',
+  message: isIntegerPercentage,
+})
+
```

Comment:
This is reliant upon the redirector which we need to remove, so this test should go too

```suggestion
```